STEERING-COMMITTEE.md

```diff
@@ -140,17 +107,6 @@ pass with majority of the committee supporting it.
 
```

Comment:
to be clear, it's: `> 50% of votes cast`, not of all members, right? And, abstain == no vote

Reply:
It's actually 51% of all members, the way the SC charter is currently written. Paul's PR doesn't change that.

Reply:
Yes but I think it should be changed, otherwise absent, or abstaining, members influence the vote

Reply:
I'm ok with the "at least half of the members" in the next sentence.

Reply:
^^ was w.r.t. quorum, not voting. Just to be clear.

re: voting....
- to me it need to be > 50% because if there's a tie then I think the status-quo should prevail
- we need to be clear about whether abstains and "not voting" counts towards the tally, otherwise people who choose to not vote (by not showing up, or voting to 'abstain') are in essence voting "no". Which means1) means we're allowing people who choose to step back from the process (and their duties) to influence things, and 2) there's no way for someone to say "I have no opinion and I'll go with the will of the other voters". That's why "abstain" should be the same as "not voting" and neither should count in the tally.  ie. "> 50% of people who vote"

Reply:
so, the way it is now (and is unchanged in this charter revision), a majority of SC needs to vote for any decision, and abstains are materially indisinguishable from "no" votes.  The advantage of the present system is mostly around preventing any one faction (or company, or whatever) from taking advantage of poor availability (say, during Kubecon) to pass changes without an actual majority.

I've been a part of organizations that allowed a minority to pass measures, but those organizations had very strict rules around what constituted a valid meeting in order to make that work -- rules I'll note that this charter lacks. That would be a lot of extra process for the SC.  What problem are you actually solving here?

Reply:
- I want a way for someone to vote with "no opinion" - can't do that today
- I don't want people who choose to not participate to have influence over votes
- and yes I agree we probably need a bit more in here about quorum/valid meetings.  E.g. announce votes at least week in advance, allow people to vote off-line if they can't make the meeting, etc...  but in small groups like this it's usually not an issue when there's trust and good will.  However, yes we should add those things....

Create a charter assuming the worst, but hope for the best.

Reply:
Can I suggest passing this "simple" version first, and then considering that kind of revision?  The advantage of always requiring a majority affirmative vote is that it considerably simplifies process; you can take a vote anywhere you want because you always need four "yeses".

Reply:
that's fine, we can defer a deeper chat for later